core/protos/o2control.proto

```diff
@@ -123,6 +123,7 @@ message TeardownReply {}
 message GetEnvironmentsRequest {
     bool showAll = 1;
     bool showTaskInfos = 2;
+    bool showIntegratedServices = 3;
```

Comment:
I think the flag name is unclear. Even when it's false, we fill `ingratedServicesData` with the output of `GetEnvironmentsShortData`, while the flag name gives impression we will get nothing. Could we name it for example `showDetailedIntegratedServices`, or anything conveying this meaning? A short comment might also help.

The fact that `showAll` does not imply we show task infos is another issue, but it's something that was there before.